README.md

```diff
@@ -134,6 +134,17 @@ The table below lists all the supported jobs with their links.
 |   21.  	| 688d093c-3b8d-11eb-adc1-0242ac120002 	|                     S3 bucket should allow only HTTPS requests                    	|             [aws-s3-bucket-policy-allow-https](remediation_worker/jobs/aws_s3_bucket_policy_allow_https)             	|
 |   22.  	| 09639b9d-98e8-493b-b8a4-916775a7dea9 	|            SQS queue policy should restricted access to required users            	|            [aws-sqs-queue-publicly-accessible](remediation_worker/jobs/aws_sqs_queue_publicly_accessible)            	|
 |   23.  	| 1ec4a1f2-3e08-11eb-b378-0242ac130002 	| Network ACL should restrict administration ports (3389 and 22) from public access 	| [aws-ec2-administration-ports-ingress-allowed](remediation_worker/jobs/aws_ec2_administration_ports_ingress_allowed) 	|
+|   24.   	|       5c8c263d7a550e1fb6560c39        |            EC2 instance should restrict public access to SSH port (20)            	|                            [ec2-close-port-20](remediation_worker/jobs/ec2_close_port_20)                            	|
+|   25.   	| 4823ede0-7bed-4af0-a182-81c2ada80203  |            EC2 instance should restrict public access to SSH port (5601)            	|                            [ec2-close-port-5601](remediation_worker/jobs/ec2_close_port_5601)                            |
```

Comment:
Rule Name for port 5601 is this - EC2 instance should restrict public access to Kibana port (5601) 
Correct the rule names for all the other entries as well. Also, check if you are using the correct rule name and rule id in all the readme files.

Reply:
sorry my bad... i copied and pasted from port 22.  I updated the README accordingly to individual port's README.
Thanks for catching this.